kolibri/core/assets/src/views/k-navbar/index.vue

```diff
@@ -2,6 +2,7 @@
 
   <nav class="tabs">
     <ul>
+      <!-- Can contain either tab-buttons or tab-links. -->
```

Comment:
note - I copied over these navbar changes in d0329ad because I needed them for #1922